k8s/aat/common/ccd/data-store-api.yaml

```diff
@@ -26,7 +26,7 @@ spec:
         CCD_DOCUMENT_URL_PATTERN: ^https?://(((?:api-gateway\.preprod\.dm\.reform\.hmcts\.net|(dm-store-aat\.service\.core-compute-aat|dm-store-(pr-[0-9]+|preview)\.service\.core-compute-preview)\.internal(?::\d+)?)\/documents\/[A-Za-z0-9-]+(?:\/binary)?)|((em-hrs-api-(?:aat|staging)\.service\.core-compute-aat|em-hrs-api-(pr-[0-9]+|preview)\.service\.core-compute-preview)\.internal(?::\d+)?\/hearing-recordings\/[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}\/segments\/[0-9]))
         CCD_CONDITIONAL_APIS_CASE_ASSIGNED_USER_AND_ROLES_ENABLED: true
```

Comment:
can't remember on top of head now, but we may need allow-listing here as well for deleting case roles. Worth asinf the particular operation to be called and double checking requirement for allow-listing here.